cli.js

```diff
@@ -34,6 +36,11 @@ files.forEach((file) => {
     } else {
       fs.writeFileSync(file, sorted, 'utf8')
       console.log(`${file} is sorted!`)
+      const diff = Diff.diffLines(packageJson, sorted)
+      diff.forEach((part) => {
+        const color = part.added ? 'green' : part.removed ? 'red' : 'grey'
+        process.stderr.write(part.value[color])
```

Comment:
I prefer use a small color lib like [`yoctocolors`](https://github.com/sindresorhus/yoctocolors) instead of extending `String.prototype`

Reply:
@fisker 
Thanks for the review.

> I prefer use a small color lib like yoctocolors instead 

Since `require()` was not supported for [yoctocolors](https://github.com/sindresorhus/yoctocolors), I did not introduce it, but introduced [picocolors](https://github.com/alexeyraspopov/picocolors).
Snapshot changes have also occurred with the lib changes, so we have updated it again.